jsonLoader.go

```diff
@@ -134,6 +150,10 @@ func (l *jsonReferenceLoader) LoadJSON() (interface{}, error) {
 	refToUrl := reference
 	refToUrl.GetUrl().Fragment = ""
 
+	if loader, ok := l.schemaLoaders[refToUrl.String()]; ok {
```

Comment:
I'm not entire sure if this is required now? When validating the flow is loader->loaderFactory->pool->cachedLoader->LoadJSON... but perhaps there are other paths that I don't yet have loaded into my head.

Reply:
It shouldn't be required. There should only need to be one place where the interception is done and right now I think that should be done at the loaderFactory level.